### DISCUSSION TITLE
Zzh! CC2652R Multiprotocol RF Stick firmware upgrade

### DISCUSSION THREAD

**Discussion by acaonweb** [Koenkk/zigbee2mqtt#12483] [General]
I'm thinking to buy a new Xiaomi RTCGQ14LM, which needs new version of coordinator firmware 
I have [this](https://www.tindie.com/products/electrolama/zzh-cc2652r-multiprotocol-rf-stick/) coordinator, with 20210120 fw version, and I think I've to upgrade it to [20211114](https://github.com/Koenkk/Z-Stack-firmware/tree/7c5a6da0c41855d42b5e6506e5e3b496be097ba3/coordinator/Z-Stack_3.x.0/bin).
How upgrade it? is it like flashing following the guide, like the first time, or I need to follow a different process?
There are some suggestion before doing it?

Thanx in advance.
Fabrizio

**Comment by Koenkk** [comment]
You can find the flashing instructions by clicking on your adapter here: https://www.zigbee2mqtt.io/guide/adapters/#recommended

**Reply by acaonweb** [reply]
Thanks for the link, so upgrade the firmware is like normal flashing , it's not a different procedure , right? 
I'll have to repair everything?
Thanks

**Reply by Koenkk** [reply]
Like normal flashing indeed, no repairing is needed (z2m will automatically create and restore the backup)
